create/create.js

```diff
@@ -0,0 +1,23 @@
+//imports
+import { createPost, checkAuth } from '../fetch-utils.js'; 
+
+
+//dom els
+const createFormEl = document.querySelector('.create-form');
+
+//state
+
+//events
```

Comment:
The Create Page should be a protected route - if a user isn't logged in, we should redirect them to the auth page. You can do that on 'load' here.